opal/corelib/kernel.rb

```diff
@@ -1114,4 +1114,9 @@ def to_s
   end
 
   alias untaint taint
+
+  def Rational(*args)
+    #Just a stub to let unrelated rubyspecs run.
+    nil
```

Comment:
@elia, it would be better, but then it fails with an exception, which I cannot use `fails` to filter out. I mean having it raise a NotImplementedError is no different in this case than the original NoMethodError that prompted me to add this stub. I agree, this is not pretty. The real solution is to go ahead and implement Rational, I guess :smile: